src/backend/views/components/rightAside.twig

```diff
@@ -0,0 +1,22 @@
+<div class="docs-right-aside">
+	<div class="docs-right-aside__title">
```

Comment:
remove the `Table of Contents` and its styles from this PR please. You can add just a grey rectangle in that place to present how it behaves.

Reply:
done